### PR TITLE
Melky

### PR DESCRIPTION
**Pull request by Betsimisaraka**
Hey team

Please have a look at my PR, happy to change it if need to be.

Thanks
Anita

### CODE REVIEW

File: m/melky.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Melky
+
+![Melky](melky.JPG)
+Her real name is Randrianarison Melky.
```
Comment (ndorfin) [inline m/melky.md:5]: New line between the image and paragraph please.

**Review by ndorfin — CHANGES_REQUESTED** [review]
Looks good, one small fix.

File: m/melky.md (hunk 1 above)
Comment (Betsimisaraka) [reply m/melky.md:5]: I have put new line between the image and paragraph.

**Review by ndorfin — APPROVED** [review]
Good job once again! 🚢 -it!

**Pull request merged**